src/components/right-menu/right-menu.html

```diff
@@ -46,5 +46,25 @@ <h1 [ngStyle]="{'color': stylingService.conditionalColoring.text}">
         </ion-label>
       </button>
     </ion-list>
+    <br/>
+    <br/>
+    <ion-list *ngIf="isLoggedIn && notJoinKhatms.length > 0">
+      <ion-list-header [color]="stylingService.conditionalColoring.backgroundLighter">
+        <h1 [ngStyle]="{'color': stylingService.conditionalColoring.text}">
+          {{ls.translate('Recently seen khatms')}}
+        </h1>
+        <h2>
+          {{ls.translate('(Not Join)')}}
+        </h2>
+      </ion-list-header>
+      <button *ngFor="let khatm of notJoinKhatms" ion-item
+              detail-none
+              [color]="stylingService.conditionalColoring.background"
+              (click)="openPage('khatm', khatm)">
```

Comment:
I highly recommend to use enums instead static text e.g: 'khatm'. it leads to have more efficient scale-able product